### PR TITLE
Client Registry Task Handling admin support

### PR DESCRIPTION
**Pull request by rayking29**
# Fixes or Implements [AB#13785](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/13785)

## Description

Added warning message to Blazor admin Support page.

## Testing

- [ ] Unit Tests Updated
- [ ] Functional Tests Updated
- [x] Not Required

## UI Changes

Screen Shot 2022-08-30 at 4 50 30 PM

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

### CODE REVIEW

**Review by sslaws — APPROVED** [review]
Approved although it looks like your PR contains commits from others.

**Comment by rayking29** [comment]
There are some files I merged to the feature branch from dev and forgot to push.  I have now pushed them there so they won't actually be part of the merge.

File: Apps/Admin/Client/Pages/SupportPage.razor [hunk 1]
```diff
@@ -38,6 +38,10 @@
     </MudGrid>
 </MudForm>
 
+<HgBannerFeedback Severity="@Severity.Normal" IsEnabled="HasWarning" TResetAction="MessageVerificationActions.ResetStateAction">
+    @MessageVerificationState.Value.WarningMessage
+</HgBannerFeedback>
```
Comment (MikeLyttle) [inline Apps/Admin/Client/Pages/SupportPage.razor:43]: Severity in mockup seems to be "Info" - should confirm expected appearance with @mk08sh.

**Review by MikeLyttle — APPROVED** [review]

File: Apps/Admin/Client/Pages/SupportPage.razor (hunk 1 above)
Comment (rayking29) [reply Apps/Admin/Client/Pages/SupportPage.razor:43]: Yes the mock-up shows a rather normal looking bar, not a warning looking bar, for the message. I will check with Nino and Mandad on that.

**Pull request merged**